config/simulation/docker/docker-compose-competition.yml

```
@@ -53,13 +51,14 @@ services:
                 - gpu
     tty: true # Allows the controller to send its output to Webots' console
     environment:
-      - QT_X11_NO_MITSHM=1
       - NVIDIA_VISIBLE_DEVICES=all
       - NVIDIA_DRIVER_CAPABILITIES=all
       - WEBOTS_STDOUT_REDIRECT=1
       - WEBOTS_STDERR_REDIRECT=1
     volumes:
-      - controller_data:$PROJECT_PATH/$THEIA_VOLUME:rw
+      - /dev/dri:/dev/dri:rw
+      - /tmp/webots-$PORT/ipc:/tmp/webots-$PORT/ipc:rw
```

I believe we should be more restrictive here to avoid that a participant connects to another robot with an extern controller.
```suggestion
      - /tmp/webots-$PORT/ipc/participant:/tmp/webots-$PORT/ipc/participant:rw
```